src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '../utils/mixins';
+
+.stars-area {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-top: 16px;
+
+  &__text {
+    @include product-code-font;
+
+    margin-top: -1px;
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+
+    &:nth-child(-n + 4),
+    &--4 {
+      background-image: url(/src/images/star-active.svg);
```

Comment:
The `:nth-child(-n + 4)` selector is not clear in terms of the BEM methodology. It's better to use a modifier class like `stars__star--active` to apply styles to active stars. This makes the HTML more readable and the CSS more maintainable. If you need to style a specific number of stars as active, consider adding a modifier to the parent block (e.g., `stars--4-active`) and style the children accordingly.